index.html

```diff
@@ -68,16 +68,12 @@ <h3>Backgrounds</h3>
         </div>
         <form class="save-outfit" action="#" method="#">
           <input id="saveOutfitInput" class="outfit-namer"type="text" name="outfitName" value="" placeholder="name this outfit">
-          <button id="saveOutfitBtn" class="button-style disable-save" type="button" name="saveOutfit">Save</button>
+          <button id="saveOutfitBtn" class="button-style disable-save" type="button" name="saveOutfit" disabled>Save</button>
         </form>
       </section>
       <section class="column3 body-column">
         <h3>Saved outfits</h3>
         <section class="outfits-list">
-          <button class="button-style">Fancy<img src="assets/close.svg" alt="Close"></button>
-          <button class="button-style">A Day Out<img src="assets/close.svg" alt="Close"></button>
-          <button class="button-style">Beach Time<img src="assets/close.svg" alt="Close"></button>
-          <button class="button-style">Mars Prom<img src="assets/close.svg" alt="Close"></button>
```

Comment:
I removed the placeholder cards now that the save card functionality is working.